src/locales/en-US/list.ftl

```diff
@@ -196,25 +185,24 @@ modal-delete = Delete this Entry?
 connect-another-device = Connect another device
 easily-access-logins = Easily gain access to your logins from any device.
 access-on-another-computer = Access on another computer
-# TODO should Firefox Account or Firefox Lockbox be localized in the following strings?
-simply-sign-in-other-device = Simply sign in to your Firefox Account on your other device to sync your logins to that computer.
+simply-sign-in-other-device = Simply sign in to your { -product-accounts } on your other device to sync your logins to that computer.
 download-mobile = Download the mobile app
-download-ios-android = Firefox Lockbox is available on both iOS and Android. <learnmore>Click here</learnmore> to learn more and to send a link to your phone to download the app.
-before-access = Before you can access your logins on another device, you will need to connect a Firefox Account.
-connect-a-firefox-account = Connect a Firefox Account
-connect-a-firefox-account-complete = Connect a Firefox Account (complete)
-sync-requires-account = To sync your logins to another device, you will need to <signin>sign in or create a Firefox Account</signin>.
+download-ios-android = { -product-full-name } is available on both iOS and Android. <learnmore>Click here</learnmore> to learn more and to send a link to your phone to download the app.
+before-access = Before you can access your logins on another device, you will need to connect a { -product-full-name }.
+connect-a-firefox-account = Connect a { -product-full-name }
+connect-a-firefox-account-complete = Connect a { -product-full-name } (complete)
```

Comment:
What's the difference between these two?

Reply:
The former is displayed in a checklist when the user has an action take, and the latter is displayed when the user has completed that checklist action